polly/lib/External/isl/imath/tests/gmp-compat-test/gendata.py

```diff
@@ -42,7 +43,7 @@ def apply(fun, lst):
 mm_all = mm_slong + mm_ulong + mm_sint + mm_uint + mm_sshort + mm_ushort
 zero_one_all = mzero_one + zero_one
 
-mpz_std_list = zero_one_all + mm_all + apply(plus1, mm_all) + apply(minus1, mm_all)
```

Comment:
Please do not modifiy code from external projects. These scripts are not used within Polly anyway.